tests/longhaul/block_on_k8s_test.go

```diff
@@ -4,6 +4,7 @@ import (
 	"sync"
 	"testing"
 
+	"github.com/coreos/pkg/capnslog"
```

Comment:
all our tests are for k8s now, so i'd suggest you just name this file `block_test.go`